subworkflows/local/frameshift_correction.nf

```diff
@@ -0,0 +1,74 @@
+include { PROKKA as PROKKA_BEFORE } from '../../modules/local/prokka'
+include { PROKKA as PROKKA_AFTER  } from '../../modules/local/prokka'
+include { DIAMOND_MAKEDB        } from '../../modules/nf-core/diamond/makedb/main'
+include { DIAMOND_BLASTP as DIAMOND_BLASTP_BEFORE } from '../../modules/nf-core/diamond/blastp/main'
+include { DIAMOND_BLASTP as DIAMOND_BLASTP_AFTER } from '../../modules/nf-core/diamond/blastp/main'
+include { IDEEL as IDEEL_BEFORE } from '../../modules/local/ideel'
+include { IDEEL as IDEEL_AFTER  } from '../../modules/local/ideel'
+include { PROOVFRAME_FIX        } from '../../modules/nf-core/proovframe/fix/main'
+include { PROOVFRAME_MAP        } from '../../modules/nf-core/proovframe/map/main'
+
+workflow FRAMESHIFT_CORRECTION {
+    take:
+    contigs                   // [ val(meta), path(contigs) ]
+
+    main:
+    ch_versions = Channel.empty()
+
+    PROKKA_BEFORE(
+        contigs
+    )
+    ch_versions = ch_versions.mix(PROKKA_BEFORE.out.versions)
+
+    DIAMOND_MAKEDB(
+        [[ id:'diamond_db' ], file(params.diamond_db, checkIfExists: true) ], [], [], []
+    )
+    ch_versions = ch_versions.mix(DIAMOND_MAKEDB.out.versions)
+
+    DIAMOND_BLASTP_BEFORE(
+      PROKKA_BEFORE.out.prokka_faa,
```

Comment:
I think we can run "pyprodigal" instead of Prokka (which is much slower because it does more)